src/floats.jl

```diff
@@ -441,15 +446,21 @@ pow10(::Type{BigFloat}, e) = (@inbounds v = F64_SHORT_POWERS[e+1]; return v)
 _unsigned(x::BigInt) = x
 _unsigned(x) = unsigned(x)
 
-function scale(::Type{T}, FT::FloatType, v, exp, neg, f::F) where {T, F}
-    if T === Float16 || FT == FLOAT16
-        return handlef(__scale(Float16, _unsigned(v), exp, neg), f)
-    elseif T === Float32 || FT == FLOAT32
+@inline function scale(::Type{T}, FT::FloatType, v, exp, neg, f::F) where {T, F}
```

Comment:
I just stepped out of the debugger and was about to say that scale prob needs to be inline as well :+1:

Reply:
👍